functions/pipeline/shared/db_access/db_access_v2.py

```diff
@@ -39,10 +38,11 @@ def __init__(self, image_id, x_min, x_max, y_min, y_max, classification_names):
 
 # Vott tags have image height & width data as well.
 class VottImageTag(ImageTag):
-    def __init__(self, image_id, x_min, x_max, y_min, y_max, classification_names, image_height, image_width):
+    def __init__(self, image_id, x_min, x_max, y_min, y_max, classification_names, image_height, image_width, image_location):
```

Comment:
@samiyaakhtar I don't think we should overload this. The training scripts have a different need. They need the box and image confidence scores. This class's usage seems to be for generating VOTT json that is needed on download. I suggest renaming this class or creating another class with all these fields plus box and image confidence. Really the word "VOTT" shouldn't exist anywhere in our db data access code cc @mtarng